cli/fmt.rs

```diff
@@ -274,50 +309,6 @@ fn write_file_contents(
   Ok(fs::write(file_path, file_text)?)
 }
 
-pub async fn run_parallelized<F>(
-  file_paths: Vec<PathBuf>,
-  f: F,
-) -> Result<(), ErrBox>
-where
-  F: FnOnce(PathBuf) -> Result<(), ErrBox> + Send + 'static + Clone,
-{
-  let handles = file_paths.iter().map(|file_path| {
-    let f = f.clone();
-    let file_path = file_path.clone();
-    tokio::task::spawn_blocking(move || f(file_path))
-  });
-  let join_results = futures::future::join_all(handles).await;
-
-  // find the tasks that panicked and let the user know which files
-  let panic_file_paths = join_results
-    .iter()
-    .enumerate()
-    .filter_map(|(i, join_result)| {
-      join_result
-        .as_ref()
-        .err()
-        .map(|_| file_paths[i].to_string_lossy())
-    })
-    .collect::<Vec<_>>();
-  if !panic_file_paths.is_empty() {
-    panic!("Panic formatting: {}", panic_file_paths.join(", "))
```

Comment:
Worth noting that this panic functionality is gone... so if something panics it won't say what file it panicked on. I could implement this again by using `catch_unwind`. Let me know if I should.

Reply:
@dsherret I think it would quite useful to keep this functionality a little longer, it helped us track quite a few bugs in recent days.